### PR TITLE
Introduce RHEL based image

### PR DESCRIPTION
**Pull request by riuvshin**
Enable build on RHEL if TARGET=rhel.

The cico environment will continue to build the CentOS based container and in a second step it will build and push a RHEL based container.

cc: @jmelis @msrb @miteshvp

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,11 @@
-REGISTRY?=registry.devshift.net
+ifeq ($(TARGET),rhel)
+  DOCKERFILE := Dockerfile.rhel
+  REGISTRY := push.registry.devshift.net/osio-prod
+else
+  DOCKERFILE := Dockerfile
+  REGISTRY := registry.devshift.net
```
Comment (jmelis) [inline Makefile]: REGISTRY sould be push.registry.devshift.net

**Review by jmelis — CHANGES_REQUESTED** [review]
I think REGISTRY should be push.registry.devshift.net

File: Makefile (hunk 1 above)
Comment (riuvshin) [reply Makefile]: nice, catch! thanks

**Review by miteshvp — APPROVED** [review]
LGTM

**Pull request merged**